update/pkg/yum/info_test.go

```diff
@@ -337,3 +337,14 @@ func TestParseInfo(t *testing.T) {
 		assert.Equal(t, time.Date(2021, 6, 1, 15, 30, 45, 0, time.UTC), releasetime)
 	})
 }
+
+func TestGetRHELVersion(t *testing.T) {
+	t.Run("getRHELVersion", func(t *testing.T) {
+		// TODO enable once we pass to RHEL 9 for good
+		t.Skip("This test will fail on RHEL 7 and non-RHEL systems")
```

Comment:
This will only pass on RHEL9
```
[root@pmm-managed-server update]# go test -timeout 30s -v -run TestGetRHELVersion/getRHELVersion ./pkg/yum/.
=== RUN   TestGetRHELVersion
=== RUN   TestGetRHELVersion/getRHELVersion
    info_test.go:348: 
                Error Trace:    /root/go/src/github.com/percona/pmm/update/pkg/yum/info_test.go:348
                Error:          Not equal: 
                                expected: "9"
                                actual  : "7"
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -9
                                +7
                Test:           TestGetRHELVersion/getRHELVersion
--- FAIL: TestGetRHELVersion (0.03s)
    --- FAIL: TestGetRHELVersion/getRHELVersion (0.03s)
```